com.adobe.Flash-Player-Projector.appdata.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2018 Bastien Nocera <hadess@hadess.net> -->
+<component type="desktop">
+  <id>com.adobe.Flash-Player-Projector</id>
+  <name>Adobe Flash Player</name>
+  <summary>Player for content created using Adobe Flash</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <!-- license linked from https://www.adobe.com/legal/licenses-terms.html -->
+  <project_license>LicenseRef-proprietary=https://wwwimages2.adobe.com/content/dam/acom/en/legal/licenses-terms/pdf/Flash_Player_30_0.pdf</project_license>
+  <launchable type="desktop-id">com.adobe.Flash-Player-Projector.desktop/</launchable>
+  <content_rating type="oars-1.1"/>
+  <description>
+    <p>
+      The Adobe Flash Player Projector is a player for content created on the Adobe Flash platform.
+    </p>
+  </description>
+  <url type="homepage">https://www.adobe.com/support/flashplayer/debug_downloads.html</url>
+  <screenshots>
+    <!-- FIXME this will need to point to a real screenshot -->
+    <screenshot type="default" width="354" height="234">https://a.fsdn.com/con/app/proj/teoemulator/screenshots/teo.png/max/max/1</screenshot>
```

Comment:
Before this PR will be accepted, you will actually have to add a screenshot (you can put it in the git repo).